srcpkgs/SmartSim/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'SmartSim'
+pkgname=SmartSim
+version=1.4
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool pkg-config"
```

Comment:
automake and libtool are normally not required when not regenerating `configure`